potodo/_github.py

```diff
@@ -1,32 +1,48 @@
+import re
 import requests
 
-from subprocess import getoutput
+from typing import Mapping
+from subprocess import check_output
 
 
-def get_repo_url():
-    url = getoutput("git remote get-url --all upstream")
+def get_repo_url() -> str:
+    """Tries to get the repository url from git commands
+    """
+    url: str = check_output(
+        "git remote get-url --all upstream", universal_newlines=True
+    )
     if "fatal" in url:
-        url = getoutput("git remote get-url --all origin")
+        url = check_output("git remote get-url --all origin", universal_newlines=True)
     if "fatal" in url:
+        # If the commands didn't work
         raise ValueError(
             f"Unknown error. `git get-url --all upstream|origin` returned {url}"
         )
     return url
 
 
-def get_repo_name():
+def get_repo_name() -> str:
+    """Will get the repository url from git commands then remove useless stuff to get ORG/NAME
+    """
     repo_url: str = get_repo_url()
```

Comment:
Typically here, as `get_repo_url` is typed, I don't feed you have to type repo_url. Check by removing the `: str` and running `mypy --strict` I bet it won't whine.